reactor-netty-http/src/test/java/reactor/netty/http/HttpDecoderSpecTest.java

```diff
@@ -32,6 +32,8 @@ void init() {
 
 	@Test
 	void maxInitialLineLength() {
+		assertThat(conf.maxInitialLineLength()).as("default initial line length").isEqualTo(HttpDecoderSpec.DEFAULT_MAX_INITIAL_LINE_LENGTH);
```

Comment:
NB: if we want to assert the _content_ of the constants, we can chain a `.isEqualTo(literal)` (or `isTrue()`/`isFalse()` for booleans). that way we assert the default is the constant AND a change to the codebase that modifies the constant's value would get flagged by unit test